lib/broccoli/translation-reducer/linter/find-missing-translations.js

```diff
@@ -1,7 +1,13 @@
-function findMissingTranslations(key, localeKeys, requiresTranslation) {
-  return localeKeys
-    .filter(([locale, translationKeys]) => !translationKeys.includes(key) && requiresTranslation(key, locale))
-    .map(([locale]) => locale);
+function findLocalesMissingTranslation(key, localeKeys) {
+  const result = localeKeys
```

Comment:
Same here

Reply:
Same as above, makes debugging a little easier so will keep as-is